adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/V3.kt

```diff
@@ -73,6 +75,16 @@ class V3 : Application() {
         )
     }
 
+    private fun schedulePeriodicUpdates() {
+        // Eagerly fetch repo from db on app startup
+        val disabled = System.getProperty(DISABLE_PERIODIC_UPDATES, "false")!!.toBoolean()
```

Comment:
I wonder if it's worth inverting this property to the positive `ENABLE_PERIODIC_UPDATES` with a default of `true` to avoid the double negative?